### DISCUSSION TITLE
希望能增加分开多次运行单个功能的选项

### DISCUSSION THREAD

**Discussion by Rinka12138** [MaaAssistantArknights/MaaAssistantArknights#6859] [Ideas]
基建换班那里如果能做成登录后先进去收一轮基建换班收菜 然后正常进行1-7打材料或者活动关 然后再进行一次基建换班补新打的土和换班就好了

**Comment by gsxgmpkt9020** [comment]
MaaX项目应该行

https://github.com/MaaAssistantArknights/MaaX

**Comment by status102** [comment]
v5.x会考虑加入这个功能

**Comment by Daydreamer114** [comment]
working on #10474
